### PR TITLE
postgresql@15 15.0 (new formula)

### PR DESCRIPTION
**Pull request by ankane**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

Notes

- Copied from `postgresql@14`
- The audit wanted this marked as `keg_only :versioned_formula`, but I'm guessing we want to make `postgresql@14` the keg-only one

### CODE REVIEW

**Comment by ankane** [comment]
There was a [`Missing libraries` error](https://github.com/Homebrew/homebrew-core/actions/runs/3257390340/jobs/5348655629) with `--libdir=#{HOMEBREW_PREFIX}/lib/#{name}` (possibly because it's marked keg-only), so the latest commit uses the directory structure from `postgresql@13`.

However, it sounds like there may be a better way to handle this from the discussion in #107726.

File: Formula/postgresql@15.rb
Comment (SMillerDev) [inline Formula/postgresql@15.rb:46]:
```suggestion
    args = std_configure_args + %W[
```

**Comment by MikeMcQuaid** [comment]
> [@SMillerDev](/SMillerDev) [SMillerDev](/SMillerDev) assigned [MikeMcQuaid](/MikeMcQuaid) [yesterday](#event-7596278330)

@SMillerDev any context for this 😅?

**Comment by SMillerDev** [comment]
That was meant to be a review request since you've been working on postgresql recently 😅

**Review by MikeMcQuaid — DISMISSED** [review]
LGTM once @SMillerDev's tweak lands.

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks so much for your repeated contribution! Without people like you submitting PRs we couldn't run this project. You rock, @ankane!

**Comment by ankane** [comment]
Thanks @MikeMcQuaid, and thanks for leading the project all these years!!

**Pull request closed without merging**
